<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>google_maps_earth_view</class>
 <widget class="QWidget" name="google_maps_earth_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/icons/icons/icons8-Traffic Jam-80.png</normaloff>:/icons/icons/icons8-Traffic Jam-80.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="google_maps_earth_layout">
   <item>
    <layout class="QHBoxLayout" name="controls_layout" stretch="0,10,0,10,0,30,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="latitude_lbl">
       <property name="text">
        <string>Latitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="latitude_edit">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="longitude_lbl">
       <property name="text">
        <string>Longitude:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="longitude_edit">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="go_btn">
       <property name="text">
        <string>Go</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/icons/icons/icons8-Map Pin-16.png</normaloff>:/icons/icons/icons8-Map Pin-16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="left_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="overlap_lbl">
       <property name="text">
        <string>Overlap:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="overlap_spn">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interval_lbl">
       <property name="text">
        <string>Interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="interval_spn">
       <property name="suffix">
        <string> sec</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="right_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/icons/icons/icons8-Play-16.png</normaloff>:/icons/icons/icons8-Play-16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
